frontend/src/views/StatblockEditorView.vue

```diff
@@ -686,6 +705,155 @@ export default defineComponent({
 				this.showSlides(moveToSlide);
 			}
 		},
+		calculateCR(): void{
+			var defenseCR: number = 0
+			var offenseCR: number = 0
+			const hp = this.data.defenses.hp.override ? this.data.defenses.hp.override : this.hpCalc()
+			console.log("HP: " + hp)
+
+			Object.keys(ChallengeRatingTable).forEach((k) => {
```

Comment:
I'll update the loop to be a for loop.

The fetch again, the loop is more to get the key I need, and from there because the fetch is more of a dict it is a O(1) so shouldn't be any different if I just set a variable in the loop to it as well. Just saves me from setting an additional variable when I know the key to get what I need is all.

Reply:
It shouldn't be any different no, but I don't see a reason to have to set a variable within the loop if it can just be done in the iterator itself. Seems cleaner to me, at least.